console_backend/src/tabs/advanced_tab/advanced_imu_tab.rs

```diff
@@ -97,18 +98,25 @@ impl AdvancedImuTab {
     /// # Parameters
     /// - `msg`: MsgImuAux to extract data from.
     pub fn handle_imu_aux(&mut self, msg: MsgImuAux) {
-        match msg.imu_type {
-            0 => {
-                self.imu_temp = 23_f64 + msg.temp as f64 / 512_f64;
+        match msg.imu_type() {
+            Ok(imu) => {
+                match imu {
+                    ImuType::BoschBmi160 => {
+                        self.imu_temp = 23_f64 + msg.temp as f64 / 512_f64;
+                    }
+                    ImuType::StMicroelectronicsAsm330Llh => {
+                        self.imu_temp = 25_f64 + msg.temp as f64 / 256_f64;
+                    }
+                    ImuType::MurataScha634D03 => {
+                        self.imu_temp = 25_f64 + msg.temp as f64 / 256_f64;
```

Comment:
for these `imu_temp` values, how should they be handled? (compared to the above)

Reply:
@silverjam they seem to have different imu_temp handling formulas (not sure how they were derived)

Reply:
cc @woodfell ^^

@adrian-kong -- if it's not documented to be different in the SBP manual, I'd leave it the same as the others